pulp_ansible/app/tasks/collections.py

```diff
@@ -385,6 +384,7 @@ def __init__(self, remote, repository, optimize):
         self._unpaginated_collection_metadata = None
         self._unpaginated_collection_version_metadata = None
         self.last_synced_metadata_time = None
+        self.page_size = settings.get("SYNC_PAGE_SIZE", 1000)
```

Comment:
Do you set this on galaxy's side?

Reply:
it defaults to 1000 but can be set to other value via PULP_SYNC_PAGE_SIZE variable